app/controllers/reasons/ChangeUnderpaymentReasonController.scala

```diff
@@ -117,8 +117,31 @@ class ChangeUnderpaymentReasonController @Inject()(identify: IdentifierAction,
       )
     )
 
-    SummaryList(itemNumberSummaryListRow ++ originalAmountSummaryListRow)
+    if (underpaymentReason.boxNumber == 99) {
```

Comment:
99 is used in multiple places, could it not be a final variable something like otherBoxNumber so it's a bit more readable than 99

Reply:
created a separate ticket for it: https://jira.tools.tax.service.gov.uk/browse/CIVDT-1367